R/roxygenize-setup.R

```diff
@@ -23,6 +25,11 @@ roxygen_setup <- function(path = ".", cur_version = NULL) {
   man_path <- file.path(path, "man")
   dir.create(man_path, recursive = TRUE, showWarnings = FALSE)
 
+  local_options(
+    "roxygen2:::package" = desc::desc_get("Package", path),
```

Comment:
Maybe we should make this an env var so it also solves #1517?

Reply:
Now done